src/content/docs/new-relic-control/agent-control/configuration.mdx

```diff
@@ -140,52 +140,55 @@ The following snippet demonstrates installing Agent Control with OpenTelemetry a
   >
 
     ```yaml
-        global:
-          cluster: "YOUR_CLUSTER_NAME"
-          licenseKey: "YOUR_LICENSE_KEY"
-        # Values related to the agent control's Helm chart release.
-        # `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
-        agent-control-deployment:
-          identityClientId: ""
-          identityClientSecret: ""
-          config:
-            fleet_control:
-              # Optional: Specify a fleet_id (entity guid) to automatically connect to an existing fleet.
-              fleet_id: ""
-              auth:
-                # New Relic organization ID where agent will connect to.
-               organizationId: "YOUR_ORGANIZATION_ID"
-          
-            # List of managed agents that will be deployed. The key represents the name of the agent and the value holds the configuration.
-           subAgents:
-             infrastructure:
-               type: newrelic/com.newrelic.infrastructure:0.1.0
-               content:
-                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle%60 
-                  # Recommended: check and define an explicit chart version (latest stable)
-                 chart_version: "*"
-             agent-operator:
-               type: newrelic/com.newrelic.k8s_agent_operator:0.1.0
-               content:
-                 chart_version: "*"
-             fluentbit:
-               type: newrelic/io.fluentbit:0.1.0
-               content:
-                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging`
-                  # Recommended: check and define an explicit chart version (latest stable)
-                 chart_version: "*"
-                 chart_values:
-                   global:
-                     lowDataMode: true
-             prometheus:
-               type: newrelic/com.newrelic.prometheus:0.1.0
-               content:
-                 chart_version: "*"
-                 chart_values:
-                   global:
-                     lowDataMode: true
-                   newrelic-prometheus-agent:
-                     config.kubernetes.integrations_filter.enabled: false
+    global:
+      cluster: "YOUR_CLUSTER_NAME"
+      licenseKey: "YOUR_LICENSE_KEY"
+    # Values related to the agent control's Helm chart release.
+    # `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
+    agent-control-deployment:
+      identityClientId: ""
```

Comment:
Should we specify here that these 2 fields are required or the chart won't install?